pi/async_server.py

```diff
@@ -133,7 +133,10 @@ async def _parse(self):
                 self.rov_state.set_target_velocity(
                     utils.VelocityVector(json.loads(json_msg["target_velocity"]))
                 )
-            
+
+            if "imu_data" in json_msg:
```

Comment:
Does this just print the output, or is it actually stored anywhere with this?

Reply:
i wrote this in the pull request as well, but for now it just gets printed. since there's no functioning frontend at the moment, there's nowhere else to send the data to, so the only thing to do is print it.